product_docs/docs/pgd/5/rel_notes/pgd_5.5.0_rel_notes.mdx

```diff
@@ -0,0 +1,87 @@
+|-----------|---------|------------------------------------------------------------------------------------------------|-----------|
+| BDR       | 5.5.0   | Added support for read-only proxy routing.                                                     |           |
+| BDR       | 5.5.0   | Improve stability of routing leader selection by using  Raft heartbeat for connectivity check. |           |
+| CLI       | 5.5.0   | Added PGD CLI binaries for macOS.                                                              |           |
+| Proxy     | 5.5.0   | Added support for read-only proxy routing.                                                     |           |
+
+
+## Enhancements
+
+| Component | Version | Description                                                                                                                    | Addresses      |
+|-----------|---------|--------------------------------------------------------------------------------------------------------------------------------|----------------|
+| BDR       | 5.5.0   | Improved bulk INSERT/UPDATE/DELETE performance by sending multiple messages together in a group rather than individually.      |                |
+| BDR       | 5.5.0   | Changes received by the writer as now not saved to a temporary file.                                                          |                |
+| BDR       | 5.5.0   | BDR now logs completion of an extension upgrade.                                                                               |                |
+| BDR       | 5.5.0   | Added restrictions for group commit options.                                                                                   |                |
+| BDR       | 5.5.0   | Each autopartition task is now executed in its own transaction.                                                                | RT101407/35476 |
+| BDR       | 5.5.0   | DETACH CONCURRENTLY is now used to drop partitions.                                                                            | RT101407/35476 |
+| BDR       | 5.5.0   | Node group creation on a node bad state is now disallowed.                                                                     |                |
+| BDR       | 5.5.0   | Granted additional object permissions to role `bdr_read_all_stats`.                                                            |                |
+| BDR       | 5.5.0   | Improve stability of manager worker and Raft consensus by not throwing error on non-fatal dynamic shared memory read failures. |                |
+| BDR       | 5.5.0   | Improved stability of Raft consensus and workers by handling dynamic shared memory errors in the right place.                  |                |
+| BDR       | 5.5.0   | The number of changes processed by writer in a large transaction is now exposed.                                               |                |
```

Comment:
@djw-m can you link to the page with more detail about what this looks like now?

Reply:
Again, no, because this release note came with no accompanying notes (and the note doesn't say where the changes are exposed).